### PR TITLE
[ADD] added skeleton, set up modules and libraries , reused redirecti…

### PR DESCRIPTION
**Pull request by EugeneTeu**
- added TS
- added redirection for authentication logic
- config files and dev environment 
- Chose to not use custom webpack , just sticking with CRA's webpack config 
- Css modules
- using ant design framework

### CODE REVIEW

File: frontend/src/pages/login/index.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+const Login : FC<{}> = () => {
+
+    return <Card className={s['loginCard']}>
+        <h1>Group Buy</h1>
+        <Button type='primary'>Google Login</Button>
+        
+
+       
+    </Card>
```
Comment (pohlinwei) [inline frontend/src/pages/login/index.tsx]: Is the indentation intended? It seems a little off to me. 😅

**Review by pohlinwei — COMMENTED** [review]
I suppose we are using 4 spaces for tabs?

File: frontend/src/pages/login/index.tsx (hunk 1 above)
Comment (EugeneTeu) [reply frontend/src/pages/login/index.tsx]: ah i fixed this!

**Comment by EugeneTeu** [comment]
Yes, Im using eslint to help me enforce

**Review by leafgecko — APPROVED** [review]

**Pull request merged**